lib/hunter/commands/parse.rb

```diff
@@ -124,18 +135,6 @@ def label_exists?(label)
         @used_strings.any? { |n| n == label }
       end
 
-      def generate_label(prefix)
-        start = @options.start
```

Comment:
It looks like we aren't using `@options.start` any more. If this is correct, could you remove references to it from this command file and `lib/hunter/cli.rb` please?